pages/common/docker-service.md

```diff
@@ -0,0 +1,28 @@
+# docker service
+
+> Manage services on docker daemon.
+> More information: <https://docs.docker.com/engine/reference/commandline/service/>.
+
+- List services on docker daemon:
+
+`docker service ls`
+
+- Create a new service:
+
+`docker service create --name {{service_name}} {{docker_image_name}}:{{docker_image_version}}`
+
+- Display detailed information on one or more space separated services:
+
+`docker service inspect {{service_name|ID}}`
+
+- List the tasks of one or more space separated services:
+
+`docker service ps {{service_name|ID}}`
+
+- Scale one or more space separated replicated services:
```

Comment:
I'm not sure about `list of services` because has the replica number:
```suggestion
- Scale to a specific number of replicas for a space-separated list of services:
```

Reply:
@navarroaxel, Yes. It is `list of services` only.

Reply:
I'm talking about the example description in our side. Maybe another reviewer will suggest a better text.

Reply:
Got it. Sure 👍